docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Migrating from Expo Firebase to React Native Firebase
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+The `expo-firebase-analytics` and `expo-firebase-recaptcha` packages allowed the use of some Firebase features on Classic Builds, Expo Go, and Expo for Web, but developers often encountered version conflicts when trying to use them alongside the rest of the Firebase suite. With EAS Build and Development Builds, it is much easier to use [React Native Firebase](https://rnfirebase.io/) directly in your entire development workflow, so these packages have been deprecated.
+
+In this guide, we will migrate from the Expo Firebase packages to React Native Firebase by:
+- Replacing the Expo Firebase packages with their React Native Firebase equivalents
+- Replacing the use of Expo Go for testing with a development build customized to your project's needs
+- Testing the updated code on the development build
+
+> **Note:** These instructions assume you have already been successfully using the Expo Firebase packages. If you're adding Firebase to your app for the first time, follow our [Using Firebase](/guides/using-firebase) guide.
+
+## Install React Native Firebase
+Run the following commands to remove `expo-firebase-analytics` and install the React Native Firebase packages:
+
+<Terminal cmd={[
+'# Remove Expo Firebase packages',
+'$ yarn remove expo-firebase-analytics',
+'',
+'# Install RN Firebase, etc.',
+'$ yarn add @react-native-firebase/app @react-native-firebase/analytics'
+]} />
+
+If you are using [Expo Prebuild](/guides/adopting-prebuild) (also known as Managed Workflow) and building your app for iOS, install the `expo-build-properties` plugin and apply the following configuration to your [Expo config]() file.
+
+<Terminal cmd={["$ npx expo install expo-build-properties"]} />
+
+#### Example app.json with config plugin
+
+```json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-build-properties",
+        {
+          "ios": {
+            "useFrameworks": "static"
+          }
+        }
+      ]
+    ]
+  }
+}
+```
```

Comment:
```suggestion
### Add Config plugin to Expo config file

In the Expo config file, add add `expo-community-flipper` to the `plugins` array with the following configuration:

```json app.json
{
  "expo": {
    "plugins": [
      [
        "expo-build-properties",
        {
          "ios": {
            "useFrameworks": "static"
          }
        }
      ]
    ]
  }
}
```
```

This seems like a step rather than an example.